man/mono.1

```diff
@@ -65,6 +65,45 @@ and
 See the description for 
 .B MONO_IOMAP
 in the environment variables section for more details.
+.SH METHOD DESCRIPTIONS
+A number of diagnostic command line options take as argument a method
+description.   A method description is a textual representation that
+can be used to uniquely identify a method.   The syntax is as follows:
+.nf
+[namespace]classname:methodname[(arguments)]
+.fi
+.PP
+The values in brackets are optional, like the namespace and the
+arguments.   The arguments themselves are either empty, or a
+comma-separated list of arguments.   Both the
+\fBclassname\fR
+and
+\fBmethodname\fR
+can be set to the special value '*' to match any values (Unix shell
+users should escape the argument to avoid the shell interpreting
+this).
+.PP
+The arguments, if present should be a comma separated list of types
+either a full typename, or for built-in types it should use the
+low-level ILAsm type names for the built-in types,
+like 'void', 'char', 'bool', 'byte', 'sbyte', 'uint16', 'int16', 'uint',
+'int', 'ulong', 'long', 'uintptr', 'intptr', 'single', 'double', 'string', 'object'.
+.PP
+Pointer types should be the name of the type, followed by a '*',
+arrays should be the typename followed by '[' one or more commas (to
+indicate the rank of the array), and ']'.
+.PP
+Generic values should use '<', one or more type names, separated by
+both a comma and a space and '>'.
+.PP
+By-reference arguments should include a "&" after the typename.
+.PP
+Examples:
+.nf
+*:ctor(int)		// All constructors that take an int as an argument
+*:Main			// Methods named Main in any class
+*:Main(string[])	// Methods named Main that take a string array in any class
```

Comment:
Add a generic example?

Reply:
eh, also no pointer example or class::method, the text is very clear as to the form, so examples are just being really exceedingly nice, plenty good enough.